extensions/campeonato-brasileiro/.eslintrc.json

```
@@ -0,0 +1,14 @@
+{
+  "root": true,
+  "env": {
+    "es2020": true,
+    "node": true
+  },
+  "parser": "@typescript-eslint/parser",
+  "plugins": ["@typescript-eslint"],
+  "extends": [
+    "eslint:recommended",
+    "plugin:@typescript-eslint/recommended",
+    "prettier"
+  ]
+}
```

```suggestion
{
  "root": true,
  "extends": ["@raycast"]
}
```
In order to make your extension the most compatible as possible with Raycast, please use this rules extension that the Raycast team recommended.

---

I did this, but when I ran `npm run lint`, I got this error:

> error -  run ESLint
Oops! Something went wrong! :(

ESLint: 7.32.0

ESLint couldn't find the config "@raycast" to extend from. Please check that the name of the config is correct.

The config "@raycast" was referenced from the config file in "/Users/danielsequeira/workspace/danielsequeira-extensions/extensions/campeonato-brasileiro/.eslintrc.json".

If you still have problems, please stop by https://eslint.org/chat/help to chat with the team.


That is why I just reverted to those other plugins.

---

Yes, great catch @danielsequeira !
You need to install this dependency: `@raycast/eslint-config` by `npm i -D @raycast/eslint-config`
Apologies for that! 🙏